SystemSettings.php

```diff
@@ -53,11 +53,20 @@ class SystemSettings extends \Piwik\Settings\Plugin\SystemSettings
     public $numRequestsToProcess;
 
     /** @var Setting */
-    public $useSentinelBackend;
```

Comment:
@haristku You have deleted this option, what if someone has already checked the option "Enable Redis Sentinel" ?
![Screenshot from 2024-10-09 08-12-59](https://github.com/user-attachments/assets/2bcbc545-3579-4a73-b526-4b098c3f0af3)